internal/common/foraging/deerhunt.go

```diff
@@ -66,6 +66,10 @@ func (d DeerHunt) getPopulationLinkedProbability(dhConf config.DeerHuntConfig, p
 	thetaCritical := dhConf.ThetaCritical
 	thetaMax := dhConf.ThetaMax
 	p := float64(population) / float64(dhConf.MaxDeerPerHunt)
+
+	if pMax == pCritical { // catch the special case where max deer per hunt = max population. You've got params wrong if you're here though :)
```

Comment:
let's log this so they know :D

Reply:
(you can pass in the server's logger func into `DeerHunt`)

Reply:
I was thinking the same. Just spam the logs but it also might be too much spam? tbf, its already spammy af so feel free to add more.

Reply:
spammy is fine as long as it is _easily filterable_

Reply:
Okay sure - makes sense!

Reply:
feel free to do this in a future PR--i'm happy with this as long as you add the extra check in `parseConfig`.